fvm/environment/account_info.go

```diff
@@ -127,3 +131,26 @@ func (info *AccountInfo) GetAccountAvailableBalance(
 	}
 	return result.ToGoValue().(uint64), nil
 }
+
+func (info *AccountInfo) GetAccount(
+	address flow.Address,
+) (
+	*flow.Account,
+	error,
+) {
+	account, err := info.accounts.Get(address)
+	if err != nil {
+		return nil, err
+	}
+
+	if info.serviceAccountEnabled {
+		balance, err := info.GetAccountBalance(common.Address(address))
```

Comment:
Note for the future: Not sure if we want to keep the balance field on the account result long term. Its probably worth discussing in TPM if this could be removed...